src/App/Pages/singleAccomodation.js

```diff
@@ -156,8 +158,8 @@ export class singleAccomodations extends Component {
 								<Link to={`/accommodations/${id}/book`} className='btn btn-primary btn-lg'>
 									<span> </span> Book it<span> </span>
 								</Link>
-								<button className='borger border-dark btn ml-auto btn-lg'>bookmark</button>
 							</div>
+							{token && <LikeAndBookMark id={id} token={token} />}
```

Comment:
I think that even though a user is not logged in he can still view the number of likes of accommodation.  he just won't be able to like it.

Reply:
Thanks, I'm waiting for backend's PR to be merged